between random: 20647777
between random: 20647777
between random: 20647771
between random: 20647777
between random: 20647777
between random: 21583
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [9,229,85,53,184,69,89,249]
between random: [26,72,90,218,64,3,125,193]
between random: [26,72,90,218,64,3,125,193]
sed: can't read /usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_frame.ivy: No such file or directory
""
/usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_frame.ivy: line 1468: error: assumption failed
output directory: temp/1885
implementation directory: /home/chris/TVOQE_UPGRADE_27/quic/aioquic/
implementation command: python3 examples/http3_server.py --quic-log /home/chris/qlog_aiquic2 --certificate /home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_cert.pem --private-key /home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_key.pem  -v --host 127.0.0.1 --port 4443 -l /home/chris/secret.log
../quic_server_test_newcoid_rtp_error (0) ...
implementation command: ['python3', 'examples/http3_server.py', '--quic-log', '/home/chris/qlog_aiquic2', '--certificate', '/home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_cert.pem', '--private-key', '/home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_key.pem', '-v', '--host', '127.0.0.1', '--port', '4443', '-l', '/home/chris/secret.log']
quic_process pid: 27757
timeout 100 ./build/quic_server_test_newcoid_rtp_error seed=780 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
